pontoon/base/static/css/table.css

```diff
@@ -130,6 +130,14 @@ table.table {
   color: #F36;
 }
 
+.table td.code div{
+  width: 70px;
+  display: block;
```

Comment:
I think we don't need `display: block;`, because the DIV element is a [block-level element](https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements).

Reply:
Ah I see, thanks for the info! And it's done ;)